network/monuments.csv

```diff
@@ -1,5 +1,5 @@
 Mark,Domes Number,Mark Type,Type,Ground Relationship,Foundation Type,Foundation Depth,Start Date,End Date,Bedrock,Geology
-2004,,Forced Centering,Steel Mast,0,Reinforced Concrete,0,2000-06-19T00:00:00Z,9999-01-01T00:00:00Z,,
+2004,,Forced Centering,Steel Mast,-1.35,Reinforced Concrete,0,2022-10-26T00:00:00Z,9999-01-01T00:00:00Z,,
```

Comment:
I think the monument height should be 0 in this case, since the geodetic mark is on the ground?

Reply:
But what type of the mark type should we use in this case?
Shoud we use different Start Date (2000-06-19T00:00:00Z) to be aligned with the mark.csv file?

Reply:
oh good question!! definitively not `steel mast`, you're right. 

and sorry I don't have an answer... currently allowed options in delta to describe a monument are:
```
Mark Type,Type
Forced Centering,Pillar
Forced Centering,Shallow Rod / Braced Antenna Mount
Forced Centering,Steel Mast
Forced Centering,Wyatt/Agnew Drilled-Braced
Iron Rod,Unknown
Not Specified,Steel Mast
Not Specified,Unknown
Unmarked Mark,Steel Mast
Unmarked Mark,Unknown
```


in the [LINZ database 2004 page](https://www.geodesy.linz.govt.nz/gdb/index.cgi?mode=gmap&sessionid=10160578311891670209733&code=2004&action=+setfoundmarks+updatelist&foundmarklist=2004&listaction=clear+add&mark=), this geodetic mark is described as:
Mark type: Pin
Beacon type (monument type): Not Beaconed

we currently don't have `pin` as an allowed Mark Type so not sure what we should use here.

Reply:
and in the appropriate tables.

Reply:
oh nice, thanks @ozym

Reply:
If we add the  new mark type - pin, what monument type should we use?

Reply:
I have been looking around comparing what we have in delta with descriptions in the LINZ db (did some random checks, so a deeper and more robust check might be useful). 

Maybe we can use "unkown" for now maybe? It has been used for sites like WRA1 (that was a forced centering mark type, but still something like a "metal thing in the ground".

Alternatively we can have a look at what IGS would use for similar use cases (even though IGS should only have continuous type monuments rather than survey marks)